core/src/test/kotlin/fr/sncf/osrd/utils/DummyInfra.kt

```diff
@@ -112,36 +103,36 @@ class DummyInfra : RawInfra, BlockInfra {
     }
 
     override val physicalSignals: StaticIdxSpace<PhysicalSignal>
-        get() = TODO("Not yet implemented")
+        get() = StaticIdxSpace(blockPool.size.toUInt())
     override val logicalSignals: StaticIdxSpace<LogicalSignal>
-        get() = TODO("Not yet implemented")
+        get() = StaticIdxSpace(blockPool.size.toUInt())
 
     override fun getLogicalSignals(signal: PhysicalSignalId): StaticIdxList<LogicalSignal> {
-        TODO("Not yet implemented")
+        return makeIndexList(signal)
     }
 
     override fun getPhysicalSignal(signal: LogicalSignalId): PhysicalSignalId {
-        TODO("Not yet implemented")
+        return convertId(signal)
     }
 
     override fun getPhysicalSignalName(signal: PhysicalSignalId): String? {
         TODO("Not yet implemented")
     }
 
     override fun getSignalSightDistance(signal: PhysicalSignalId): Distance {
-        TODO("Not yet implemented")
+        return 400.meters
     }
 
     override fun getSignalingSystemId(signal: LogicalSignalId): String {
-        TODO("Not yet implemented")
+        return "BAL"
```

Comment:
This doesn't seem to be really used, but I'm a little worried that it would cause issues later on. But we don't have an easy way to get the name from the signaling system id.

Maybe add a comment to explain why it always returns BAL? Or we could implement a getter for a signaling system name from its id.

Reply:
We don’t have any physical signals or logical signals implemented in Dummy infra.
leaving a comment for now

Reply:
If it wasn't needed we'd keep the `TODO("Not yet implemented")`, but it seems that it's actually used.

We don't have signal data here yet but the signal and block data are strongly intertwined, it's not immediately obvious that it's not required, even without signal pools.

Reply:
as discussed, now returning the type of signaling on the block